sdk/src/oauth/state-machines/debug-oauth-2026-07-28.ts

```
@@ -1311,14 +1343,59 @@ export const createDebugOAuthStateMachine = (
             // finding, not something to paper over). This binds the metadata
             // document to the issuer and is the anchor every later issuer check
             // (record_issuer, callback `iss`) trusts.
+            // A same-origin path-prefix "mismatch" is the multi-tenant AS shape
+            // (issuer at the origin root, endpoints scoped under a path). Only
+            // the per-server "Path-scoped authorization server" opt-in accepts
+            // it — mirroring the XAA debugger's toggle.
+            let pathScopedIssuerAccepted = false;
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)